libraries/libfc/secp256k1/CMakeLists.txt

```diff
@@ -5,10 +5,12 @@ add_library(secp256k1-internal INTERFACE)
 target_include_directories(secp256k1-internal INTERFACE secp256k1/src)
 
 target_compile_definitions(secp256k1-internal INTERFACE ENABLE_MODULE_RECOVERY=1
-                                                        ECMULT_GEN_PREC_BITS=4
+                                                        ENABLE_MODULE_EXTRAKEYS=1
+                                                        COMB_BLOCKS=11
+                                                        COMB_TEETH=6
```

Comment:
These changes are snooped out via
https://github.com/bitcoin-core/secp256k1/blob/e3a885d42a7800c1ccebad94ad1e2b82c4df5c65/CMakeLists.txt#L104-L119

I needed to add `ENABLE_MODULE_EXTRAKEYS` to get the tests to compile. That might be an oversight in the upstream project (`MODULE_EXTRAKEYS` is required by `MODULE_SCHNORRSIG` -- both enabled by default in default config -- so it may be uncommon to compile without `MODULE_EXTRAKEYS`). Doesn't really matter though: just some tiny bit of unused-by-spring code added.